### DISCUSSION TITLE
記事を公開するボタンが「公開設定」になっているのはわかりにくいのでは？

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#523] [要望・アイデア・不具合]
https://github.com/increments/qiita-discussions/discussions/511 と一緒にリクエストすべきだったかもしれませんが、いつかのアップデートから記事を公開するボタンのラベルが「公開設定」になりました。

<img width="1053" alt="Screenshot 2023-08-17 at 13 55 45" src="https://github.com/increments/qiita-discussions/assets/1148320/c5be450b-0172-45c5-9817-3378a5e5de22">

実際、ボタンをクリックしたら設定画面が開いて、そこから実際の更新を実行するわけですが、

<img width="1023" alt="Screenshot 2023-08-17 at 13 58 14" src="https://github.com/increments/qiita-discussions/assets/1148320/1a9ce019-246c-4a30-94fb-d6697ced727f">

書き手の心理としては「よし書けた！公開するぞ〜！・・・・公開する・・・あれ、公開するボタンは？？」となってしまいます。
「公開設定」の向こうに「公開する」ボタンがあるのは記事の入力フォームを見ても想像が付きにくいです。
（設定ボタンは設定をするためだけのボタンに見える）

https://github.com/increments/qiita-discussions/discussions/511 の改善で緑色の目立つボタンになったので、多少は「もしかしてこれ？」と気づけるようになったかもしれませんが、それでも自信をもって「これだ！」とは言いにくいUIな気がします。

## 改善案

ボタンのラベルを「公開設定」から「公開する」「更新する」のようなラベルに変える。
ただし、ボタンをクリックしてもまだ続きがありそうなデザインにする。
例： `公開する >>` 、`公開する →` etc

**Comment by tsubolabo** [comment]
https://qiita.com/tobatiie/items/a0c22882573027b55f92

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます🙏

ボタンの文言ですが、ボタン位置を変えた際に一緒に変更しておりました。
https://qiita.com/release-notes/2023-07-11

JunichiItoさんがご記載くださった、書き手の方の心理もとても共感できます。
一方で、noteも同様の形式になっており、現在の文言で違和感がないといったご意見もいただいています。
また、「公開設定」を「投稿する」に変えることで、どこが「公開設定」なのかわからなくなってしまう懸念もあります。

あまり頻繁に変更が重なると逆に混乱を招きそうなため、しばらく様子を見て
様々なご意見を参考に、今後の改善方法を検討したいと思います🙇

JunichiItoさんにはご不便をおかけすることになり申し訳ないのですが
ご理解いただけますと幸いです🙏

**Reply by JunichiIto** [reply]
@tanaka-asami415 返信ありがとうございます。承知しました。
